modules/custom-status/custom-status.php

```diff
@@ -1239,6 +1239,27 @@ public function workflow_is_publish_blocked( $post_id ) {
 		}
 	}
 
+
+	/**
+	 * Given a post ID, return true if the post type is supported and using a custom status, false otherwise.
+	 *
+	 * @param int $post_id The post ID being queried.
+	 * @return bool True if the post is using a custom status, false otherwise.
+	 */
+	public function is_post_using_custom_status( $post_id ) {
+		$post = get_post( $post_id );
+
+		if ( null === $post ) {
+			return false;
+		}
+
+		$custom_post_types = $this->get_post_types_for_module();
```

Comment:
Minor: We should rename this function because it's not module specific anymore.

Reply:
True! Changed to `get_supported_post_types()` in https://github.com/Automattic/vip-workflow-plugin/pull/19/commits/f792715e4aa735a2a8119e77fe429f1b5931854e.